Release of API version 1.0.0-rc.1 (#121)
camaraproject/NumberVerification: README.md

## Meetings

* Current schedule, registration, & meeting links are available on the confluence page: [Meetings information](https://wiki.camaraproject.org/display/CAM/NumberVerification)

## Status and released versions

* Note: Please be aware that the project will have frequent updates to the main branch. There are no compatibility guarantees associated with code in any branch, including main, until a new release is created. For example, changes may be reverted before a release is created. **For best results, use the latest available release**.

> * **The latest available release and version of CAMARA OTPValidaton API is 1.0.0-rc.1 This is the first stable version candidate of the API**. It is suitable for implementors and could be use with customers in productive environments.

Reviewer: ```suggestion
* **The latest available release and version of CAMARA Number Verification API is 1.0.0-rc.1 This is the first stable version candidate of the API**. It is suitable for implementors and could be use with customers in productive environments.
```

Author: This is now fixed.

> * **The latest available release and version of CAMARA Number Verification API is 1.0.0-rc.1 This is the first stable version candidate of the API**. It is suitable for implementors and could be use with customers in productive environments.

Reviewer 2: ```suggestion
* **The latest pre-release of CAMARA Number Verification API is 1.0.0-rc.1 This is the release candidate of the first stable version of the API**. It is suitable for implementors.
```
Only the public version v1.0.0 should be used in production. The release candidate is for implementors only.
